app/uk/gov/hmrc/helpfrontend/controllers/HelpController.scala

```diff
@@ -58,15 +59,15 @@ class HelpController @Inject() (
     Ok(cookiesPage())
   }
 
-  private def maybeOverrideLang[A](newLang: Option[String])(action: Action[A]): Action[A] =
-    Action.async(action.parser) { initialRequest =>
-      Try(initialRequest.withTransientLang(newLang.get)) match {
-        case Success(changedRequest) => action(changedRequest).map(_.withLang(changedRequest.lang))
-        case Failure(_)              => action(initialRequest)
+  private def maybeChangeLang[A](newLang: Option[String])(action: Action[A]): Action[A] =
+    Action.async(action.parser) { request =>
+      newLang match {
+        case Some(lang) if List("en", "cy").contains(lang) => Future(Redirect(request.path).withLang(Lang(lang)))
```

Comment:
Very small request but Instead of hard coding "en" and "cy" we can do List(En.code, Cy.code).contains(lang)

Reply:
@kyle-bowden @ellamdav on this one instead of using En.code I think play provides a way to get a list of the available languages that get configured in application conf, we could use that instead

Reply:
yeah i was going to look at that - there's something in MessagesApi i think

Reply:
I fiddled with it then got distracted by slack, just pushed the change I tried @ellamdav @kyle-bowden what do you think? Sorry- should have asked to jump on a call and look at it together before starting fiddling

Reply:
Looks good to me